final/problem-def-ana/problem-ana.tex

```diff
@@ -14,42 +13,33 @@ \section{Problem Analysis}
 languages, without any configuration from the part of the language
 designer, is hard, and may even be impossible.
 
-The general solution to many of the problems is to identify which
-parts of the problem can be solved generically, and which parts
-warrant configuration by the language designer. When the REPL requires
-configuration by the language designer, enough flexibility should be provided
-to do so with the least amount of hassle as possible.
+The solution to many of the problems is to identify which parts of the problem
+can be solved generically, and which parts warrant configuration by the
+language designer. When the REPL requires configuration by the language
+designer, enough flexibility should be provided to do so.
 
 Each of \cref{ssec:supp-diff-ways,ssec:lang-spec-addit,ssec:repl-spec-semant}
-discusses a more concrete instance of this general problem and
-explains in more detail how the general solution applies to that
-specific instance.
+discusses a more concrete instance of this general problem.
 
 \subsection{Supporting different execution pipelines}
 \label{ssec:supp-diff-ways}
-Different languages have different ways of executing a program in that
-language. For example, some languages have an analysis step, whereas
+Different Spoofax languages have different ways of executing a program.
+For example, some languages have an analysis step, whereas
 others do not. Moreover, specifying the dynamic semantics of a language
 can be done in multiple ways (see \cref{ssec:dynamic-semantics}): one
 can use DynSem, Stratego, and some languages even implement a Java
 backend\footnote{See IceDust:
   \url{https://github.com/MetaBorgCube/IceDust}}.
 
 Some of these properties of a language should be detected by the REPL
-automatically, such as detecting whether a language has an analysis
-step.
-
+automatically, such as whether a language has an analysis step.
 Other properties of a language, however, cannot be detected
 automatically. For example, the REPL should be able to maintain an
 execution environment for the duration of the REPL session, but
 precisely what an execution environment looks like differs among
-languages. This is therefore an area where the language designer
-should configure the REPL so that it does the evaluation the way they
-want. To accommodate for this, the REPL should provide a configuration
-interface that is generic enough for the REPL to be configured
-for any language.
+languages.
 
-\subsection{Language-specific additional commands}
+\subsection{Language-specific commands}
```

Comment:
I think having `additional` here emphasizes that they are defined outside of the core backend.